### PR TITLE
Various doc patches

### PR DESCRIPTION
**Pull request by abegong**
Changes proposed in this pull request:
- See commit messages
-
-

After submitting your PR, CI checks will run and @tiny-tim-bot will check for your CLA signature.

For a PR with nontrivial changes, we review with both design-centric and code-centric lenses.

In a design review, we aim to ensure that the PR is consistent with our relationship to the open source community, with our software architecture and abstractions, and with our users' needs and expectations. That review often starts well before a PR, for example in github issues or slack, so please link to relevant conversations in notes below to help reviewers understand and approve your PR more quickly (e.g. `closes #123`).

Previous Design Review notes:
-
-
-

### Definition of Done
Please delete options that are not relevant.

- [ ] My code follows the Great Expectations [style guide](https://docs.greatexpectations.io/en/latest/contributing/style_guide.html?highlight=style%20guide)
- [ ] I have performed a [self-review](https://docs.greatexpectations.io/en/latest/contributing/contribution_checklist.html?highlight=checklist) of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added [unit tests](https://docs.greatexpectations.io/en/latest/contributing/testing.html#contributing-testing-writing-unit-tests) where applicable and made sure that new and existing tests are passing.
- [ ] I have run any local integration tests and made sure that nothing is broken.

Thank you for submitting!

### CODE REVIEW

File: docs/tutorials/getting_started/intro.md [hunk 1]
```diff
@@ -12,8 +12,6 @@ For a tutorial for older versions of Great Expectations, please see older versio
 
 The tutorial will walk you through the following steps:
 
-![minimap](../../images/minimap.png)
-
```
Comment (cdkini) [inline docs/tutorials/getting_started/intro.md:16]: Could we go ahead and delete `minimap.png` from the codebase if we no longer use it?

File: docusaurus.config.js [hunk 2]
```diff
@@ -9,7 +9,7 @@ module.exports = {
   baseUrl: "/",
   onBrokenLinks: "throw",
   onBrokenMarkdownLinks: "warn",
-  favicon: "img/favicon.ico",
```
Comment (cdkini) [inline docusaurus.config.js:12]: Similar question around the favicon.

File: sidebars.js [hunk 3]
```diff
@@ -28,13 +28,18 @@ module.exports = {
       items: [
         {
           type: 'category',
-          label: '⚙️ Setting up Great Expectations',
+          label: 'Setting up Great Expectations',
```
Comment (cdkini) [inline sidebars.js:31]: I'm not super well versed in the planned design changes we're making to docs after the initial release so I may not be the best reviewer for this particular file.

Looks good to me when I render it in the preview though!

**Review by cdkini — APPROVED** [review]
Left a few very minor comments but nothing is blocking so I'm happy to approve.

Thanks for cleaning this up 🎉

File: docs/tutorials/getting_started/intro.md (hunk 1 above)
Comment (abegong) [reply docs/tutorials/getting_started/intro.md:16]: Good catch. I've removed it.

File: docusaurus.config.js (hunk 2 above)
Comment (abegong) [reply docusaurus.config.js:12]: There actually wasn't a favicon---that's why this needed to change.

File: sidebars.js (hunk 3 above)
Comment (abegong) [reply sidebars.js:31]: I loved these little icons, but people with more *ahem* taste than me disagreed. So we're removing them.

**Pull request merged**